book/01-introduction/sections/what-is-git.asc

```diff
@@ -30,6 +30,7 @@ Git arbeitet im Großen und Ganzen eher wie ein mit einigen unglaublich mächtig
 ==== Fast jede Funktion arbeitet lokal
 
 Die meisten Aktionen in Git benötigen nur lokale Dateien und Ressourcen, um ausgeführt zu werden – im Allgemeinen werden keine Informationen von einem anderen Computer in Ihrem Netzwerk benötigt.
+// "diesen"? Bisher war von "ihm" noch nie die Rede. --> einen
 Wenn Sie an ein CVCS gewöhnt sind, in dem die meisten Operationen diesen Netzwerk-Latenz-Overhead haben, wird Sie dieser Aspekt von Git denken lassen, dass die Götter der Geschwindigkeit Git mit außerirdischen Kräften gesegnet haben.
```

Comment:
Ich bin mit dem ganzen Satz nicht ganz glücklich.
Die "Götter der Geschwindigkeit" klingen doch etwas seltsam.
Hast die einen besseren Vorschlag?

Reply:
Das klingt tatsächlich komisch, steht aber so im englischen Original und zusammen mit den "außerirdischen Kräften" ist der Satz so "verkorkst", dass es schon wieder ok zu sein scheint. Nein, leider fällt mir keine bessere Übersetzung ein.

Reply:
Das scheint eine Art sprichwörtliche, amerikanische Floskel zu sein. Mein Englisch lässt mich da in Stich, um den Sinn der Phrase in einem deutschen Spruch auszudrücken. 
Wir sollten das im Hinterkopf behalten und darauf zurückkommen, wenn uns etwas Besseres einfällt.

Reply:
OK, doch noch ein Versuch:
```suggestion
Wenn Sie an ein CVCS gewöhnt sind, bei dem die meisten Operationen mit dieser Netzwerklatenz Overhead haben, wird dieser Aspekt von Git Sie denken lassen, dass „Gottes Segen“ Git mit übernatürlichen Kräften beschenkt hat.
```

Reply:
Ist vielleicht noch keine optimale Übersetzung, aber schon deutlich besser als das Original. Ich würde allerdings nach wie vor noch das "dieser" (Overhead) in "einen" ändern.

Reply:
Lies mal den Satz mit einer keinen, Pause hinter "Netzwerklatenz" – dann bekommt er eine neue Bedeutung

Reply:
Noch etwas besser, aber ich glaube, dass die Übersetzung immer noch nicht optimal ist. Ich würde sagen, dass unser beider (Fach-)Englisch dafür einfach nicht ausreicht. Im Englischen ist aber ganz sicher "ein" Wort mit "netzwork latency overhead" gemeint.

Mein Vorschlag: Mache ein neues Issue auf und sammle dort "nicht-optimale" Übersetzungen.

Für den Moment kannst du dann deinen letzten Vorschlag annehmen. Damit wäre dann auch der ganze Pull Request fertig und alle Kommentare wären wieder weg. Einverstanden?

Reply:
Ich sehe -- warum auch immer -- gerade deinen neuesten Vorschlag mit 

> ... bei dem die meisten Operationen mit die Netzwerklatenz einen Overhead haben, ...

nicht mehr. Dieser klingt (wieder) komisch, aber wenn man das "die" weglässt, dann klingt es ganz gut. Was meinst du?

```suggestion
Wenn Sie an ein CVCS gewöhnt sind, bei dem die meisten Operationen mit Netzwerklatenz einen Overhead haben, dann wird dieser Aspekt von Git Sie glauben lassen, dass Git mit übernatürlichen Kräften von „Gottes Segen“ beschenkt wurde.
```

Reply:
Manchmal gibt es ja auch einfach keine guten Übersetzungen von Fachbegriffen. Deshalb könnte man auch noch einmal "network latency overhead" in Klammern hinter "Overhead" setzen.

Reply:
Ja, da war ich zu schnell und habe es widerrufen.
Noch ein letzter Versuch:
```
Wenn Sie mit einem CVCS vertraut sind, bei dem die meisten Operationen durch Overhead eine Netzwerk-Latenz haben, dann wird diese Eigenschaft von Git Sie glauben lassen, dass Git von „Gottes Segen“ mit übernatürlichen Kräften bedacht wurde.
```

PS: Die Idee mit den gesammelten "fragwürdigen" Übersetzungen behalte ich im Hinterkopf. Ich habe es mal im Projekt auf die ToDo-Liste gesetzt.

Reply:
Auch gut. Nimm das.